lib/all-badge-examples.js

```diff
@@ -1832,6 +1832,30 @@ const allBadgeExamples = [
           'imagelayers'
         ]
       },
+      {
+        title: 'MicroBadger Size',
+        previewUri: '/microbadger/image-size/_/httpd.svg',
+        keywords: [
+          'docker',
+          'microbadger'
+        ]
+      },
+      {
+        title: 'MicroBadger Size (tag)',
+        previewUri: '/microbadger/image-size/_/httpd/alpine.svg',
+        keywords: [
+          'docker',
+          'microbadger'
+        ]
+      },
+      {
+        title: 'MicroBadger Layers',
+        previewUri: '/microbadger/layers/_/httpd.svg',
```

Comment:
Does layers support tag as well? Perhaps it would be worth including a fourth example.